src/EditorFeatures/CSharpTest/SimplifyTypeNames/SimplifyTypeNamesTests.cs

```diff
@@ -1201,6 +1201,20 @@ static void Main(string[] args)
         var x = nameof(typeof([|Int32|]));
     }
 
+    static string nameof(Type t)
+    {
+        return string.Empty;
+    }
+}",
+@"using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        var x = nameof(typeof(int));
+    }
+
     static string nameof(Type t)
     {
         return string.Empty;
```

Comment:
this test seems to ahve been bogus.  It was trying to validate that we were not converting 'Int32' to 'int' inside of a nameof.  However, in this particular test, we don't have a real 'nameof'.  Instead, we have a method called 'nameof'.  In that case we *do* want to simplify to 'int' as it is legal here.